app/code/Magento/CatalogMessageBroker/etc/di.xml

```diff
@@ -50,4 +50,29 @@
             <argument name="batchSize" xsi:type="number">500</argument>
         </arguments>
     </type>
+
+    <type name="Magento\CatalogMessageBroker\Model\StorefrontConnector\Connector">
+        <arguments>
+            <argument name="connectionType" xsi:type="string">in-memory</argument>
```

Comment:
Its's better to configure via an environment variable, e.g. 
            <argument name="connectionType" 
xsi:type="init_parameter">some\corresponding\path::SERVICE_COMMUNICATION_CONNECTION_TYPE</argument>

and in Instal.php add ability to provide connection type which will equal by default "in-memory"

So this type later can be changed without code changes, but with changes of an environment variable